source/testnet/guides/become-baker.rst

```diff
@@ -5,185 +5,89 @@
 
 .. _become-a-baker:
 
-==================================
-Become a baker (create blocks)
-==================================
+==========================================
+Become a baker using the Concordium Client
+==========================================
 
 .. contents::
    :local:
    :backlinks: none
 
-This section explains what a baker is, its role in the network and how to become
-one.
+This guide takes you through the steps involved in upgrading your node to a baker node and managing the node using the Concordium Client. 
 
-By reading this section you will learn:
+The process of becoming a baker involves the following:
 
--  What is a baker and the concepts related to it.
--  How to upgrade your node to become a baker.
-
-The process of becoming a baker can be summarized in the following steps:
-
-#. Get an account and some GTUs.
-#. Get a set of baker keys.
+#. Create account and acquire GTU.
+#. Create a set of baker keys.
 #. Register the baker keys with the account.
 #. Start the node with the baker keys.
 
 After completing these steps, the baker node will bake blocks. If a baked block
-is added to the chain the node's baker will receive a reward.
+is added to the chain, the baker of the node will receive a reward.
```

Comment:
I think perhaps it should be just "If a baked block is added to the chain, the **baker node** will receive a reward"